components/auto-complete/index.tsx

```diff
@@ -161,31 +158,29 @@ const AutoComplete: React.ForwardRefRenderFunction<RefSelectProps, AutoCompleteP
       {optionChildren}
     </Select>
   );
-};
+});
 
-const RefAutoComplete = React.forwardRef<RefSelectProps, AutoCompleteProps>(
-  AutoComplete,
-) as unknown as (<
+type AutoCompleteType = <
   ValueType = any,
   OptionType extends BaseOptionType | DefaultOptionType = DefaultOptionType,
 >(
   props: React.PropsWithChildren<AutoCompleteProps<ValueType, OptionType>> & {
     ref?: React.Ref<BaseSelectRef>;
   },
-) => React.ReactElement) & {
-  Option: typeof Option;
-  _InternalPanelDoNotUseOrYouWillBeFired: typeof PurePanel;
-};
+) => React.ReactElement;
 
 // We don't care debug panel
 /* istanbul ignore next */
-const PurePanel = genPurePanel(RefAutoComplete);
+const PurePanel = genPurePanel(InternalAutoComplete);
 
-RefAutoComplete.Option = Option;
-RefAutoComplete._InternalPanelDoNotUseOrYouWillBeFired = PurePanel;
+const AutoComplete = Object.assign(InternalAutoComplete as AutoCompleteType, {
+  Option,
+  /** @internal */
```

Comment:
Should not be `internal`, may cause type definition loss.

Reply:
> Should not be `internal`, may cause type definition loss.

Is `_InternalPanelDoNotUseOrYouWillBeFired` prepared for external users?

Reply:
Is `_InternalPanelDoNotUseOrYouWillBeFired` prepared for external users?